roles/openshift_health_checker/openshift_checks/etcd_traffic.py

```diff
@@ -0,0 +1,59 @@
+"""Module that scans journalctl for messages caused as a symptom of increased etcd traffic"""
+
+from openshift_checks import OpenShiftCheck, get_var
+
+
+class EtcdTraffic(OpenShiftCheck):
+    """Check if host is being affected by an increase in etcd traffic."""
+
+    name = "etcd_traffic"
+    tags = ["health", "etcd"]
+
+    @classmethod
+    def is_active(cls, task_vars):
+        """Skip hosts that do not have etcd or masters in their group names."""
+        master_count = get_var(task_vars, "openshift", "master", "master_count", default=1)
+        group_names = get_var(task_vars, "group_names", default=[])
+        active = "masters" in group_names
+        if int(master_count) > 1:
+            active = "etcd" in group_names
```

Comment:
In terms of code flow, it is strange that this completely overrides the previous value.

The logical expression seems to be:

```python
return super(EtcdTraffic, cls).is_active(task_vars) and (master_could > 1 and 'etcd' in group_names or 'masters' in group_names)
```

Surely complicated-looking... better to give names:

```python
return parent_is_active and (is_etcd_on_multimaster or is_single_master)
```


Now I'm unsure what the actual rule should be... could you please explain in English?